round.go

```diff
@@ -79,6 +81,7 @@ func (r Rounder) Round(c *Context, d, x *Decimal) Condition {
 
 // roundAddOne adds 1 to abs(b).
 func roundAddOne(b *big.Int, diff *int64) {
+	// REVIEW: any way we can avoid calling this twice?
```

Comment:
I'm not aware of a better way to determine if there was a 9 -> 10 rollover here which added another decimal digit. At least it uses a lookup table for up to 128 bits so it's generally constant time..